ostree-upload-server.py

```diff
@@ -213,19 +204,43 @@ def run(self):
 
         logging.info("Starting server on %d..." % self._port)
 
-        logging.debug("task completed callback %s", latest_task_complete);
+        logging.debug("task completed callback %s", latest_task_complete)
 
         def completed_callback(latest_task_complete):
-            logging.debug("task completed callback %s", latest_task_complete);
+            logging.debug("task completed callback %s", latest_task_complete)
             latest_task_complete[:] = [time()]
 
-        workers = Workers(self._repo, partial(completed_callback, latest_task_complete))
+        workers = Workers(partial(completed_callback, latest_task_complete))
         workers.start(task_list, self._workers)
 
-        def webapp_callback(task_name, filepath):
-            task_list.add_task(Task(task_name, filepath))
-
-        webapp = UploadWebApp(__name__, active_upload_counter, webapp_callback)
+        push_adapters = {}
+        config = SafeConfigParser(allow_no_value = True)
+        config.read('ostree-upload-server.conf')
+        remote_sections = [name
+                           for name in config.sections()
+                           if name.startswith('remote-')]
```

Comment:
Generators are nice but this would be much more readable/maintainable in a loop even if it's much more verbose